frontend/src/components/transactions/Transactions.tsx

```diff
@@ -36,14 +37,16 @@ const Transactions: React.FC<InnerProps> = React.memo(({ items }) => {
 
 export type Props = {
   fetch: StaticConfig<TransactionBaseInfo, TransactionPagination>["fetch"];
+  queryKey: ReactQuery.QueryKey | null;
```

Comment:
I think we can make it optional, not nullable:
```ts
queryKey?: ReactQuery.QueryKey
```
In this case we can keep `transactions` page without a change